apps/explorer/src/ui/Banner.tsx

```diff
@@ -37,10 +38,15 @@ const bannerStyles = cva(
 			border: {
 				true: '',
 			},
+			display: {
```

Comment:
Curious why we need to do this? Seems kind of weird

Reply:
To truncate the unrecognized token text in instances where the amount is very large.

Reply:
What if you just div-wrap this? I can't imagine you need to set the display on this node itself